wallets/exodus-extension/src/extension/registry.ts

```diff
@@ -0,0 +1,30 @@
+import { Wallet } from '@cosmos-kit/core';
+import { GoDesktopDownload } from 'react-icons/go';
+import { RiChromeFill } from 'react-icons/ri';
+
+import { EXODUS_ICON } from '../constants';
+
+export const exodusExtensionInfo: Wallet = {
+  name: 'exodus-extension',
+  logo: EXODUS_ICON,
```

Comment:
@liujun93 I wonder if we should standarize this, maybe we can make a build script. This is actually more efficient to use the base64 encoding, something to consider — I like that it's both in the directory of images, but also been base64 encoded. Pretty cool feature

Reply:
I used to use hex, but I can switch to base64 as you suggested. But FYI any encoding settings in wallet registry depends on the wallet not us

Reply:
totally! good catch June! I meant that it's nice that they used the encoding style. Very fast for load times. Perhaps we make a build step for this in the `wallet` folder